packages/contracts/src/dollar/facets/UbiquityPoolFacet.sol

```diff
@@ -46,6 +46,16 @@ contract UbiquityPoolFacet is IUbiquityPool, Modifiers {
         return LibUbiquityPool.collateralUsdBalance();
     }
 
+    /// @inheritdoc IUbiquityPool
+    function pause() external onlyAdmin {
```

Comment:
There's no need to update any of the existing contracts, libs or interfaces. We can simply [disable collateral](https://github.com/ubiquity/ubiquity-dollar/blob/a1b11e1b944227144d7cc19ab320220d9fce8b69/packages/contracts/src/dollar/libraries/LibUbiquityPool.sol#L1129) (as stated in the issue description) which will take the same effect as pausing.